nexus-repository-composer/pom.xml

```diff
@@ -69,6 +69,29 @@
       <version>1.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Apologies if I'm missing something, but I don't think these dependency changes/additions are required. I deleted these changes, deleted my local .m2/repository folder, and the build and tests appear happy. Could you try removing these dependency changes and see if they work for you?

Reply:
@bhamail You are right. The build goes perfectly fine without those redundant dependencies. Fixed.

Reply:
@bhamail It works fine without those dependencies. Fixed :)